### ISSUE TITLE
Is it possible to have this create a new alert whenever a new installation or uninstallation occurs? 

### ISSUE THREAD

**Issue by FlexMasterC** [IsaacGood/Monitor-Application-Changes-PowerShell#1]
It looks like the script attempts to update the Syncro Alert instead of populating a new one when this script runs if the alert was not previously cleared. So the updated date changes, but no new info is written to the alert. One way around this potentially is to automate the resolution of the alert, but the goal for us is to just use this to monitor these changes rather than create tickets for them.

**Comment by IsaacGood** [comment]
Yes, if the alert is not cleared and the script determines another notification is needed, it updates the alert as there can only be one alert per asset with the same category name. However the new information IS written to the alert but due to a bug in Syncro, depending where you look, it still shows the old value. From my bug report:

> On RMM Alerts tab the Failure Description column does not update when the same alert is retriggered, it still shows the value from the initial trigger event. Same for when you click View Alert. However when you go to the asset’s Monitoring tab the current correct value is shown.

Unfortunately Syncro support has chosen to ignore this and has closed my bug report with seemingly no intention of fixing it. So yes, you'd either need to manually clear the alerts, set up an Automated Remediation to clear them, or choose another notification method (Activity Log, Email or Ticket). I guess I could add a variable option to close the alert right after opening it. If you want to do so yourself just add `Close-Rmm-Alert -Category "Application Changes"` towards the end of the script after the alerts are created.
